test/distribution/test_mixture.py

```diff
@@ -39,7 +40,7 @@ def diff(x: NPArrayLike, y: NPArrayLike) -> np.ndarray:
     return np.mean(np.abs(x - y))
 
 
-NUM_SAMPLES = 1000
+NUM_SAMPLES = 100_000
```

Comment:
does the test still run in less than 2s?

Reply:
I made some changes and now it runs in under 2s.